service/core/client/src/main/scala/com/lightbend/lagom/internal/client/ClientServiceCallInvoker.scala

```diff
@@ -26,6 +26,7 @@ private[lagom] abstract class ClientServiceCallInvoker[Request, Response](
   val descriptor: Descriptor
   val serviceLocator: ServiceLocator
   val call: Call[Request, Response]
+  def headerFilter: HeaderFilter = descriptorHeaderFilter(descriptor)
```

Comment:
This was being called everywhere. Not bad, but `headerFilter` matches better what we need. I didn't want to make it a `lazy val` becasue of the perf penalty. A `val` could also work, but not sure about initialization order and I didn't want to take any risk (and was too lazy to investigate :-))